datafusion/core/src/execution/context/mod.rs

```diff
@@ -1306,6 +1381,12 @@ pub struct SessionState {
     table_factories: HashMap<String, Arc<dyn TableProviderFactory>>,
     /// Runtime environment
     runtime_env: Arc<RuntimeEnv>,
+    /// [FunctionFactory] to support pluggable user defined function handler.
+    /// It is invoked on `CREATE FUNCTION` and `DROP FUNCTION` statements.
+    ///
+    /// Datafusion generic SQL dialect does not support `CRETE FUNCTION` statement
```

Comment:
BTW once I release the next version of sqlparser-rs (hopefully today) this will no longer be true

Reply:
updated